Modules/System/Azure Blob Services API/src/API Enums/ABSLeaseAction.Enum.al

```diff
@@ -0,0 +1,54 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// The types of allowed lease operations
+/// See: https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api
+/// </summary>
+enum 9049 "ABS Lease Action"
+{
+    Access = Internal;
+    Extensible = false;
+
+    /// <summary>
+    /// Requests a new lease.
+    /// </summary>
+    value(0; acquire)
```

Comment:
Minor thing, but naming convention for enum values would be `Acquire` here (same applies to below values).

Reply:
I changed that, but when I tested it I noticed a small issue. In `GetLeaseActionFromRequestHeader` in `"ABS Optional Parameters"` I now needed to change the "lookup" of the Enum (because `Enum::"ABS Lease Action".Names.IndexOf` returned nothing with the lowercase value. 
I added a small change to make the first char uppercase, but I'm not sure if that's best practice. Could you maybe validate this?

Reply:
Hi! Please take a look at my new comments, which if implemented would remove GetLeaseActionFromRequestHeader function.

If we were to keep the function, I think it would be better to introduce a helper function that maps string -> lease action enum, instead of depending on the enum names directly.